cwltool/task_queue.py

```diff
@@ -0,0 +1,119 @@
+# Copyright (C) The Arvados Authors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+"""TaskQueue."""
+
+import queue
+import threading
+
+from typing import Callable, Optional
+
+from .loghandler import _logger
```

Comment:
I switched to the CWL logger, from the Arvados logger

Reply:
Thanks